spring-integration-azure/src/main/java/com/microsoft/azure/spring/integration/servicebus/queue/ServiceBusQueueTemplate.java

```diff
@@ -52,10 +51,13 @@ public boolean subscribe(String destination, @NonNull Consumer<Message<?>> consu
         subscribedQueues.add(destination);
         IQueueClient queueClient = this.senderFactory.getQueueClientCreator().apply(destination);
 
+        Function<UUID, CompletableFuture<Void>> success = queueClient::completeAsync;
+        Function<UUID, CompletableFuture<Void>> failure = queueClient::abandonAsync;
```

Comment:
Is this success/failure too simple? Or put this success/failure into another function?

Reply:
The real purpose of this is avoid explicit type conversion before. What do you mean by `simple`